cleanup_project.py

```
@@ -0,0 +1,282 @@
+            # 保護されたディレクトリをスキップ
+            dirs[:] = [d for d in dirs if not any(p.rstrip('/') in os.path.join(root, d) for p in self.protected_patterns)]
+            
+            for file in files:
+                file_path = os.path.join(root, file)
+                try:
+                    size = os.path.getsize(file_path)
+                    if size > size_mb * 1024 * 1024:
+                        size_mb_actual = size / (1024 * 1024)
+                        self.cleanup_report["large_files"].append((file_path, size_mb_actual))
+                        print(f"  🔍 大きなファイル: {file_path} ({size_mb_actual:.1f}MB)")
+                except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)